modules/@apostrophecms/schema/ui/apos/logic/AposInputRelationship.js

```diff
@@ -204,13 +206,47 @@ export default {
         : {};
 
       await this.search(qs);
+      if (this.searchList.length) {
+        // psuedo focus first element
+        this.searchFocusIndex = 0;
+      }
     },
     handleFocusOut() {
       // hide search list when click outside the input
       // timeout to execute "@select" method before
       setTimeout(() => {
         this.searchList = [];
```

Comment:
should we reset `this.searchSuggestion` and `this.searchHint` too?

Reply:
I don't think its necessary .. those props are refreshed every search and the UI is closed every selection (when it is reopened search is run again), so I don't think the get out of sync